mmdet/core/bbox/match_costs/match_cost.py

```diff
@@ -281,3 +282,66 @@ def __call__(self, mask_preds, gt_masks):
             mask_preds = mask_preds.sigmoid()
         dice_cost = self.binary_mask_dice_loss(mask_preds, gt_masks)
         return dice_cost * self.weight
+
+
+@MATCH_COST.register_module()
+class CrossEntropyLossCost:
+    """CrossEntropyLossCost.
+
+    Args:
+        weight (int | float, optional): loss weight. Defaults to 1.
+        use_sigmoid (bool, optional): Whether the prediction uses sigmoid
+                of softmax. Defaults to False.
+    Examples:
+         >>> from mmdet.core.bbox.match_costs import CrossEntropyLossCost
+         >>> import torch
+         >>> bce = CrossEntropyLossCost(use_sigmoid=True)
+         >>> cls_pred = torch.tensor([[7.6, 1.2], [-1.3, 10]])
+         >>> gt_labels = torch.tensor([[1, 1], [1, 0]])
+         >>> print(bce(cls_pred, gt_labels))
+    """
+
+    def __init__(self, weight=1., use_sigmoid=False):
+        self.weight = weight
+        self.use_sigmoid = use_sigmoid
```

Comment:
We should assert use_sigmoid=True here since the case of False is not implemented.
The assertion error should be raised as early as possible.